main.py

```diff
@@ -79,8 +79,14 @@
                     help='value by which flow will be divided. Original value is 20 but 1 with batchNorm gives good results')
 parser.add_argument('--milestones', default=[100,150,200], metavar='N', nargs='*', help='epochs at which learning rate is divided by 2')
 
+parser.add_argument(
+    "--seed-split",
```

Comment:
Is it possible to call it split-seed instead ?

More over you can place this snippet a little bit earlier in the code, just after split-value to keep consistency accross parameters